demomultistoreform/config/index.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors
```

Comment:
Could you please use the license headers used in https://github.com/PrestaShop/example-modules/blob/master/demoextendgrid/src/Controller/DemoOrderController.php ?

Then we don't need to update each year the files to update the 2020 in 2021 or 2022 😉